dataland-email-service/src/main/kotlin/org/dataland/datalandemailservice/services/EmailContactService.kt

```diff
@@ -22,23 +22,20 @@ class EmailContactService(
     @Value("\${dataland.notification.sender.name}") private val senderName: String,
 ) {
     private val logger = LoggerFactory.getLogger(javaClass)
+    private val internalContacts: List<EmailContact>
+    private val internalCcContacts: List<EmailContact>
```

Comment:
```suggestion
    private lateinit var internalCcContacts: List<EmailContact>
```

Compiler warning: Property must be initialized, be final, or be abstract